solidity/dashboard/src/services/staking-port-backer.service.js

```diff
@@ -173,7 +173,7 @@ const getDelegations = async (
 
     const delegationAmount = web3Utils.toBN(amount)
     // StakingPortBacker contract requires the delegation amount to be greater than zero.
-    if (delegationAmount.gt(0)) {
+    if (delegationAmount.gt(web3Utils.toBN(0))) {
```

Comment:
Wondering if [`n` postfix](https://github.com/indutny/bn.js/#postfixes) would work...
```suggestion
    if (delegationAmount.gtn(0)) {
```

Reply:
Yup, it works. Changed to `gtn`